bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusBinding.java

```diff
@@ -183,7 +187,20 @@ protected void internalReceiveCommand(String itemName, Command command) {
             if (provider.providesBindingFor(itemName)) {
                 ModbusBindingConfig config = provider.getConfig(itemName);
                 ModbusSlave slave = modbusSlaves.get(config.slaveName);
-                slave.executeCommand(command, config);
+                /* we need to handle rollershutter commands in a different way */
+                if (config.getItemClass() == RollershutterItem.class) {
```

Comment:
Would it be better if this logic of handling roller shutter rules would be in the slave? Now I think some commands are handled in slave.executeCommand, some here directly